manticore/native/cpu/abstractcpu.py

```diff
@@ -264,9 +264,9 @@ class Abi:
     Used for function call and system call models.
     """
 
-    def __init__(self, cpu):
+    def __init__(self, cpu: "Cpu"):
```

Comment:
Does we need the quotes around `Cpu`? Is it because `Cpu` is defined later?

Reply:
Exactly!  In those circumstances, you need to use a string to annotate the type as a "forward reference".

Reply:
Yep! Okay, just making sure we aren't putting things in quotes that don't need to be 👍